### PR TITLE
Cofense - Rebranding of PhishMe

### PR DESCRIPTION
**Pull request by orlichter1**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/17306

## Description
Rebranding of PhishMe Integration to be "Cofense".

## Does it break backward compatibility?
   - No - Made brand new integration and added "To be deprecated" to old one.

## Must have
- [ ] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

### CODE REVIEW

**Comment by orlichter1** [comment]
@yaakovi 
finally done with the circle.
this is the link to the authentication we pushed. how do we push both of them together?
https://github.com/demisto/content-test-conf/pull/472

File: Tests/conf.json [hunk 1]
```diff
@@ -1842,6 +1847,7 @@
         "McAfee NSM": "Nightly - Checking",
         "Forcepoint": "Nightly - Checking",
         "Phishme Intelligence": "Proxy has a non empty default value, will require fixing and merging",
+        "Cofense Intelligence": "Proxy has a non empty default value, will require fixing and merging",
```
Comment (yaakovi) [inline Tests/conf.json]: fix the problem instead of adding to unmockalbes.

File: Releases/LatestRelease/integration-PhishMe.md [hunk 2]
```diff
@@ -0,0 +1 @@
+To be deprecated. New version is called "Cofense Intelligence"
```
Comment (yaakovi) [inline Releases/LatestRelease/integration-PhishMe.md]:
```suggestion
Deprecated. New version is called "Cofense Intelligence"
```

File: Tests/conf.json (hunk 1 above)
Comment (orlichter1) [reply Tests/conf.json]: I think it works. Erased it from mock disable and it passed circle :)

**Comment by orlichter1** [comment]
@kirbles19 Hi Alex, could you please go over this? https://github.com/demisto/content/edit/Cofense_2/Integrations/integration-Cofense-Intelligence.yml?pr=%2Fdemisto%2Fcontent%2Fpull%2F3781
Thanks

**Comment by kirbles19** [comment]
@orlichter1 All files done.

**Pull request merged**